visualizations/frontend/other/TBigNumber.vue

```diff
@@ -1,174 +1,212 @@
-    >
-        <div class="flex items-center gap-2">
-            <div class="hidden opacity-50 md:block" v-if="hasIcon">
-                <slot name="icon"></slot>
-            </div>
-            <span class="text-xs font-semibold leading-4 tracking-widest">
-                {{ label || '--' }}
-            </span>
-            <base-tooltip v-if="tooltipText" :description="tooltipText" />
-        </div>
-        <div :class="hasTallSize ? 'mt-6' : 'mt-3'">
-            <span class="font-normal leading-8" :class="fontSizes[bigNumberSize]">
-                <div class="flex items-center gap-2">
-                    <span  :style="{ color: numberTextColor }">{{ value || '--' }}</span>
-                    {{ valueText }}
-                    <div v-if="previous">
-                        <span class="opacity-80" v-if="num(value) > num(previous)">
-                            <arrow-down-icon :size="18" v-if="isIncreaseBad" />
-                            <arrow-up-icon :size="18" v-else />
-                        </span>
-                        <span class="opacity-80" v-else-if="num(value) < num(previous)">
-                            <arrow-up-icon :size="18" v-if="isIncreaseBad" />
-                            <arrow-down-icon :size="18" v-else class="opacity-90" />
-                        </span>
-                        <div class="opacity-30" v-else>-</div>
-                    </div>
-                    <t-loading-spinner position="relative" v-if="loading" />
-                </div>
-            </span>
-        </div>
-        <div v-if="previous" class="flex items-center gap-1 mt-3">
-            <div class="font-semibold">
-                {{ previous }} {{ previousText }}
-            </div>
-            <div class="flex gap-2 opacity-80" v-if="percentage">
-                {{ percentage }} {{ percentageText }}
-            </div>
-            <div v-else class="opacity-80">
-                {{ noChangeText }}
-            </div>
-        </div>
-    </div>
+	<div
+		class="w-full lg:p-5 md:p-3 p-1 rounded-[2px] break-all relative"
+		:style="{ background: bgColor, color: textColor }"
+		:class="[hasUnderline && 'border-b-2 border-[#21214c]', customClass]"
+		@mouseleave="showTooltip = false"
+	>
+		<div class="relative flex items-center gap-2">
+			<div class="hidden opacity-50 md:block" v-if="$slots.icon">
+				<slot name="icon"></slot>
+			</div>
+			<span class="text-xs font-semibold leading-4 tracking-widest">
+				{{ label || "--" }}
+			</span>
+
+			<div v-show="$slots.tooltip">
+				<t-tooltip :width="tooltipWidth" :position="tooltipPosition" :open="showTooltip" @show="showTooltip = true">
+					<slot name="tooltip"></slot>
+				</t-tooltip>
+			</div>
+		</div>
+
+		<div class="mt-3">
+			<span class="font-normal leading-8" :class="fontSizes[bigNumberSize]">
+				<div class="flex items-center gap-2">
+					<t-tooltip position="right">
+						<span slot="trigger" :style="{ color: numberTextColor }">
```

Comment:
This was a old already merged code of Meg, re-added it to new syntax inside tooltip.